notebook/notebookapp.py

```diff
@@ -71,6 +71,7 @@
     from tornado.netutil import bind_unix_socket
 
 from notebook import (
+    JUPYTER_NOTEBOOK_TAG,
```

Comment:
Per previous comment, this can be removed.

Reply:
JUPYTER_NOTEBOOK_TAG is positive identification tag;
there are ton's of "Kevin" over the world;
there are many "Kevin Bates" over my Facebook search;
we ONLY wanted you "Kevin Bates";
remember reused pid process may also response "/api" with "version" field.

Reply:
"api/status" is an authenticated api.
kernel_request() MUST retrieve Jupyter Notebook identification tag before or after authentication.

Reply:
Thanks, @abaelhe. See my comment [here](https://github.com/jupyter/notebook/pull/5716#issuecomment-687305368).